edXRegressionAutomated/Tests/Settings/config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+class GlobalVariables(object):
```

Comment:
You should change the name of this class to something else. When I read it as GlobalVariables (in a different file), I was thinking they really are global. However, you're using them as a class (which is better anyway).